src/main/java/net/pms/formats/OGG.java

```diff
@@ -0,0 +1,27 @@
+package net.pms.formats;
+
+public class OGG extends Format {
+
+	@Override
+	public Identifier getIdentifier() {
+		return Identifier.OGG;
+	}
+
+	@Override
+	public boolean transcodable() {
+		return true;
+	}
+
+	public OGG() {
+		type = VIDEO;
+	}
+
+	@Override
+	public String[] getSupportedExtensions() {
+		return new String[] {
+			"ogg",
+			"ogm",
+			"ogv"
```

Comment:
@Sami32 Again I was tired, and I didn't expect it to be merged so quickly when I did it, but considering the circumstances I thought is was better to merge it before release than not, to avoid breaking AAC audio in most of the renderers.

I therefore didn't dwell on thing like that, I only rearranged what was already there.